modules/publication/ajax/FileUpload.php

```diff
@@ -446,26 +446,37 @@ function notify($pubID, $type) : void
     $emailData['User']        = $user->getFullname();
     $emailData['URL']         = $url . '/publication/view_project/?id='.$pubID;
     $emailData['ProjectName'] = $config->getSetting('prefix');
-
-    $sendTo = isset($_POST['notifyLead']) && $_POST['notifyLead'] === 'true'
-        ? array($data['LeadInvestigatorEmail']) : [];
-    // get collaborators to notify
-    $collaborators = isset($_POST['collaborators'])
-        ? json_decode($_POST['collaborators'], true) : [];
-
-    foreach ($collaborators as $c) {
-        if ($c['notify']) {
-            $sendTo[] = $c['email'];
-        }
-    }
-    if (!empty($sendTo)) {
-        $sendTo = implode(', ', $sendTo);
-        Email::send(
-            $sendTo,
-            "notifier_publication_$type.tpl",
-            $emailData
+        $Notifier     = new \NDB_Notifier(
+            "publication",
+            $type
         );
-    }
+            $msg_data = [
```

Comment:
This block looks like it's indented wrong.. it randomly goes up an indent level with no change in scope. I'm not sure why phpcs isn't flagging it.